gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/BuildEnvironmentBundleTask.java

```diff
@@ -66,23 +71,38 @@ DirectoryProperty getConfigFolder() {
     public void perform() {
         final EnvironmentBundleCreator environmentBundleCreator = getInstance(EnvironmentBundleCreator.class);
         final List<File> metaDataFiles = collectFiles(into.getAsFile().get().getPath(), YML_EXTENSION);
-        if(metaDataFiles.isEmpty()) {
+        if (metaDataFiles.isEmpty()) {
             throw new MissingEnvironmentException("Metadata file does not exist.");
         }
+
         metaDataFiles.stream().forEach(metaDataFile-> {
             final Pair<String, Map<String, String>> bundleEnvironmentValues = environmentConfigurationUtils.parseBundleMetadata(metaDataFile, configFolder.getAsFile().get());
             if (null != bundleEnvironmentValues) {
-                final String bundleFileName = bundleEnvironmentValues.getLeft() + "." + configName.get() + ".environment.bundle";
+                final String envBundleFilename = getEnvBundleFileName(bundleEnvironmentValues.getLeft());
```

Comment:
As we are setting default value as EMPTY for configName, change this to 
final String envBundleFilename  = bundleEnvironmentValues.getLeft() + "-" + configName.get()+ "env" + INSTALL_BUNDLE_EXTENSION;

Reply:
@RaghuReddy there is actually some processing we need. This name generator is moved to another method. I will update the Javadoc for that.